tests/AstrolabeWorkloadExecutor/workload-executor.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp1.1;net452</TargetFrameworks>
+    <RootNamespace>WorkloadExecutor</RootNamespace>
+    <LangVersion>8.0</LangVersion>
+    <Nullable>enable</Nullable>
```

Comment:
This ONLY affects the workload executor. As this is a completely separate console application,  it seems like a reasonable time to try out this feature: it could help us develop some empathy for our users who are using our driver with this featured enabled. I'm happy to remove this if this is controversial in any way.

Some related info:
https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/nullable-reference-types

Reply:
Discussed offline: I will remove line 8.